package.json

```diff
@@ -4,7 +4,7 @@
   "description": "The build toolchain against JavaScript Fatigue",
   "main": "./src/shared/index.js",
   "scripts": {
-    "test": "yarn run lint",
+    "test": "yarn run jest",
```

Comment:
so `yarn` is a dependence of `vitamin`?
wouldn't be better to start test with `./node_modules/.bin/jest …`